app/src/main/res/values/dimens.xml

```diff
@@ -46,4 +46,6 @@
     <dimen name="font_huge">24sp</dimen>
 
     <!-- Add your custom dimens here -->
+    <!-- Login Dimens-->
```

Comment:
this is almost perfect. You are in dimens file, is not necessary to "tell" to the label that those are login dimens, is just `<!-- Login -->`